### ISSUE TITLE
Tune NON_JAVA_BUILDS

### ISSUE THREAD

**Issue by RichardExley** [oracle/microservices-datadriven#82]
Some ideas:
1. Try running in parallel
2. yum is slow for python on nodejs builds.  Maybe download instead (as with inventory-go build)

**Comment by RichardExley** [comment]
Tried running in parallel today for Java and Polyglot builds.  Both speeded up quite a lot but order-helicon build gave errors.  We don't need instability.  Here is the error I saw:

`[ERROR] Failed to execute goal on project order-helidon: Could not resolve dependencies for project io.helidon.applications:order-helidon:jar:0.0.1-SNAPSHOT: Failed to collect dependencies at io.helidon.microprofile.bundles:helidon-microprofile:jar:2.2.0 -> io.helidon.microprofile.bundles:helidon-microprofile-core:jar:2.2.0 -> io.helidon.microprofile.server:helidon-microprofile-server:jar:2.2.0 -> io.helidon.microprofile.cdi:helidon-microprofile-cdi:jar:2.2.0 -> io.helidon.config:helidon-config-mp:jar:2.2.0: Failed to read artifact descriptor for io.helidon.config:helidon-config-mp:jar:2.2.0: Could not transfer artifact io.helidon.config:helidon-config-mp:pom:2.2.0 from/to central (https://repo.maven.apache.org/maven2): /home/ll5915_use/.m2/repository/io/helidon/config/helidon-config-mp/2.2.0/helidon-config-mp-2.2.0.pom.part (No such file or directory) -> [Help 1]`
